src/Plugin_Command.php

```diff
@@ -1148,13 +1152,35 @@ public function uninstall( $args, $assoc_args = array() ) {
 				}
 			}
 
-			if ( ! Utils\get_flag_value( $assoc_args, 'skip-delete' ) && $this->delete_plugin( $plugin ) ) {
-				WP_CLI::log( "Uninstalled and deleted '$plugin->name' plugin." );
+			if ( ! Utils\get_flag_value( $assoc_args, 'skip-delete' ) ) {
+				if ( $this->delete_plugin( $plugin ) ) {
+					$deleted_plugin_files[] = $plugin->file;
+					WP_CLI::log( "Uninstalled and deleted '$plugin->name' plugin." );
```

Comment:
This logic branch didn't appear to be covered before (uninstall success and delete failure). I don't know how to reproduce it using the purely spec driven tests as it appears to need a mock